parquet-hadoop/src/main/java/org/apache/parquet/hadoop/metadata/ColumnChunkMetaData.java

```diff
@@ -566,7 +567,7 @@ public Statistics getStatistics() {
   private final ParquetMetadataConverter parquetMetadataConverter;
   private final byte[] encryptedMetadata;
   private final byte[] columnKeyMetadata;
-  private final InternalFileDecryptor fileDecryptor; 
```

Comment:
This could be changed by IDE, but it is unnecessary.

Reply:
I'll undo them